host/class/uac/usb_host_uac/include/usb/uac_host.h

```diff
@@ -81,14 +82,14 @@ typedef void (*uac_host_device_event_cb_t)(uac_host_device_handle_t uac_device_h
 /**
  * @brief  USB UAC host class descriptor print callback
  *
- * @param[in] desc  Pointer to the USB configuration descriptor
- * @param[in] class  Class of the UAC device
- * @param[in] subclass  Subclass of the UAC device
- * @param[in] protocol  Protocol of the UAC device
+ * @param[in] desc            Pointer to the USB configuration descriptor
+ * @param[in] iface_class     Interface Class of the UAC device
+ * @param[in] iface_subclass  Interface Subclass of the UAC device
+ * @param[in] protocol        Protocol of the UAC device
  *
  */
 typedef void (*print_class_descriptor_with_context_cb)(const usb_standard_desc_t *desc,
-        uint8_t class, uint8_t subclass, uint8_t protocol);
+        uint8_t iface_class, uint8_t iface_subclass, uint8_t protocol);
```

Comment:
Sometimes, when we need to provide couple of parameters in the callback (usually, more then two), it is better to use the structure pointer. 

If you need to add some parameters later, you don't have to change the callback, just add more members, which could be easier. 

```suggestion
        iface_params_t *iface_params);
```

where: 

```
typedef struct {
    uint8_t bNum;
    uint8_t bClass;
    uint8_t bSubclass; 
    uint8_t bProtocol;
    // all other things
} iface_params_t;
```

Reply:
Thx for the suggestion, yes makes much sense to use dedicated struct. I just wanted to make least possible changes. But I guess, I can update it to struct :slightly_smiling_face: